packages/client/src/__tests__/dmmf.test.ts

```diff
@@ -347,23 +347,22 @@ describe('dmmf', () => {
     try {
       await getDMMF({ datamodel })
     } catch (e) {
-      const snapshot = `Get DMMF: Schema parsing
-error: Error validating: You defined the enum \`PostKind\`. But the current connector does not support enums.
-  -->  schema.prisma:14
-   | 
-13 | 
-14 |       enum PostKind {
-15 |         NICE
-16 |         AWESOME
-17 |       }
-   | 
+      expect(stripAnsi(e.message)).toMatchInlineSnapshot(`
+        Get DMMF: Schema parsing
+        error: Error validating: You defined the enum \`PostKind\`. But the current connector does not support enums.
+          -->  schema.prisma:14
+           | 
+        13 | 
+        14 |       enum PostKind {
+        15 |         NICE
+        16 |         AWESOME
+        17 |       }
+           | 
 
-Validation Error Count: 1`
-      if (process.env.PRISMA_CLI_QUERY_ENGINE_TYPE === 'binary') {
-        expect(stripAnsi(e.message)).toEqual(`${snapshot}\n`)
-      } else {
-        expect(stripAnsi(e.message)).toEqual(snapshot)
-      }
```

Comment:
Where did this "logic" go?
(And why was it there in the first place, and maybe why is it not needed any more now?)

Reply:
No longer needed as now the output from the binary is the same as the library (at least in terms of error messages). Previously, the binary had an additional newline